rebench/tests/persistency_test.py

```diff
@@ -168,3 +174,58 @@ def _exec_rebench_db(self, cmd_config, server):
 
         run = list(cnf.get_runs())[0]
         run.close_files()
+
+    def test_check_file_lines(self):
+        ds = DataStore(self.ui)
+        cnf = Configurator(load_config(self._path + '/persistency.conf'),
+        ds, self.ui, data_file=self._tmp_file)
+        ds.load_data(None, False)
+        ex = Executor(cnf.get_runs(), False, self.ui)
+        ex.execute()
+        current_line = 0
+        with open(self._tmp_file, 'r') as file:
+            for line in file:
```

Comment:
You may simply want to read all lines of the file into a list with 
`lines = file.readlines()`
see:
https://docs.python.org/3/library/io.html?highlight=readlines#io.IOBase.readlines

Then the following code also becomes simpler, because you just need to test the lines directly instead of having to count and check them